packages/wrangler/src/dev.tsx

```diff
@@ -497,9 +497,13 @@ export async function startApiDev(args: StartDevOptions) {
 			configParam
 		);
 
+		//if args.bundle is on, don't disable bundling
+		//if there's no args.bundle, and configParam.no_bundle is on, disable bundling
+		//otherwise, enable bundling
+		const disableBundling = !(args.bundle ?? !configParam.no_bundle);
```

Comment:
How about:

```suggestion
		const enableBundling = args.bundle ?? !configParam.no_bundle;
```

Then lower down:

```
noBundle: !enableBundling
```